sae_training/sparse_autoencoder.py

```diff
@@ -138,7 +147,14 @@ def forward(self, x: torch.Tensor, dead_neuron_mask: torch.Tensor | None = None)
         l1_loss = self.l1_coefficient * sparsity
         loss = mse_loss + l1_loss + mse_loss_ghost_resid
 
-        return sae_out, feature_acts, loss, mse_loss, l1_loss, mse_loss_ghost_resid
+        return ForwardOutput(
```

Comment:
This should be equivalent to returning a plain tuple, since `NamedTuple` extends tuple. This is just so we can also access the fields by name if we want to make sure we don't accidentally mix up which loss goes in which position when reading or writing from this tuple

Reply:
Love it!